### DISCUSSION TITLE
NBMiner v40.1

### DISCUSSION THREAD

**Discussion by nebutech-admin** [NebuTech/NBMiner#657] [General]
- `fix`: `ethash` v40.0 false detected as LHR lock under lastest Nvidia driver versions.
- `feature`: supports future LHR GPU models.

<hr /><em>This discussion was created from the release <a href='https://github.com/NebuTech/NBMiner/releases/tag/v40.1'>NBMiner v40.1</a>.</em>

**Comment by m1k3b13n** [comment]
@nebutech-admin works like a charm, both platforms good to go with nvidia overclocks applied in bat, using 0 to ignore the amd's in the mix. thank you!

also, so far working better with latest nvidia drivers at eth hashrate compared to 39.7 (newest drivers caused hashrate to drop by ~0.5-8mhs per card, this version is sitting at higher lhr unlock value at same running duration). and already noticing less hash jumping.

**Reply by ghost** [reply]
My 3060 Dual LHR cards won't stop on chopping on my HashRate.

How do I get the 39.7 NBMiner back in my system, it was working great then.

It doesn't matter which release of the NiceHash I install, I can go back and still everything is with the new updated 40.1 NBMiner!!

40.1 might use less energy but the 39.7 was more stable.

**Comment by M0TFF** [comment]
Can anyone tell me the best setting to use with my card please?
I will be mining ETH.

I have an Nvidia GeForce RTX 3080 Ti LHR
Driver version 496.76

Thanks for any help.

**Reply by Aras35** [reply]
Core 1350 mem 3000 pl 0

**Reply by Faltskog** [reply]
> Core 1350 mem 3000 pl 0

????????????????????????????????!?!?!?!?!

**Comment by Jumboninja99** [comment]
I am using 2 GPUs both RTX 3090, both are mounted on mother board and both slots are PCIe 4.0 x16 slots. both are same Make and Model.
Tried Overclocking with MSI afterburner and also with NBMiner Overclock setting. 
No matter how hard I try, Core Clock speed of GPU2 (i.e. device 1) drops below 1000, resulting in lower hashrate.

Does it need further Overclock at developers end or what??
![NBMiner](https://user-images.githubusercontent.com/94979956/143266132-21a9297e-2d15-4e7a-9bfb-fd1be9141add.png)

**Reply by sterlingfx** [reply]
Your ID1-3090 with the 110.5MH is showing MTmp 110 (C), so that could possibly be the reason for your throttling? cuz thermal issues to vram?

**Reply by Jumboninja99** [reply]
yeah, right

**Reply by Faltskog** [reply]
MSI Afterburner should be use only for tests purpouses, not overall... you have to undervolt your GPU's and fix a core clock speed and memory clock.

But responding to your question, maybe is silicon lottery, hard pill to swallow because of the amount of difference between to exact same cards, but i could be and that's the way it is...

**Comment by m1k3b13n** [comment]
it looks like its throttling bc of the temp. try to cool it and see if hash
bumps

**Comment by SekretTikhon** [comment]
Hello.
Help me solve the problem, please.
Problem:
LHR unlocking occurs in about 50% of miner launches.
Given:
rtx 3060 LHR from Gigabytes
windows (personal computer, putting another OS is not an option)
driver 472.12 (tried several different ones, didn't help)
nbminer version 40.1 (the problem is observed in earlier versions of the miner)
Manual setting of LHR1 or LHR2 mode does not help
The problem occurs randomly from startup to startup. It may start immediately successfully, or 3 - 4 miner restarts may be required.
I use a nice hash miner to switch between algorithms when one of them (including ETH) is more profitable. And therefore, once successfully launching and forgetting about the problem is not an option.
If necessary, I can provide more information and / or logs, just write what additional information is needed.
UPD: after the miner on the ETH algorithm starts successfully, all subsequent restarts on the ETH algorithm are also successful. It turns out that the problem is observed only if the nbminer was started on the ETH algorithm immediately after the nbminer was just stopped on another algorithm (kawpow or octopus). Could it be that the miner does not have time to free resources from his last launch?
![Снимок экрана (125)](https://user-images.githubusercontent.com/30767224/143334431-dc9c62a0-5aea-4318-9258-c1c33ee215fc.png)
![Снимок экрана (126)](https://user-images.githubusercontent.com/30767224/143334437-5b92160a-6714-4d98-b652-09b00c9b45ec.png)
![Снимок экрана (127)](https://user-images.githubusercontent.com/30767224/143334444-dbd26df4-5533-4b18-89f5-5742bde84c2c.png)

**Reply by Vicenturi** [reply]
Same problem here. with 2x 3080 LHR I get 71mh when LHR unlok works fine, but most of the time it ends in 40mh, even when the console shows that LHR mode is 71.

The worst part about this is that NBminer doesnt notice the low hasrate and keep minning with ethash instead of switching to othe algo.

**Reply by MaceUmbrella** [reply]
@Vicenturi are you using the correct startup arguments? i think the arguments changed with one of the recent updates, 39.7 i believe. Should be "--lhr-mode 01" (no quotes obviously). You should be getting LHR = 74, not 71, unless alot of locks are detected, it steps down till less locks are detected.

I dont know why you are getting 40mh on a 3080. Screenshots of client showing temps/clocks?

**Reply by Vicenturi** [reply]
Didn't know arguments have changed, will try what you say, thanks. Can update screenshots later, but mem temps are 86-88 in one GPU and 88-92 in the other. My usual OC parameters for best performance with all algos are CC +200, MC +1300, PL 68, but have tried lowering  Both clocks and got the same issue.

**Reply by Vicenturi** [reply]
Tried those arguments and its working fine FTM, thanks @MaceUmbrella . Miner started with ethash and it used to always work fine in first launch. Will let it mine and see how it does after mining other algo and come back to ethash.

**Reply by SekretTikhon** [reply]
I tried to turn on lhr-mod-1 and watched it work for several days.
This will solve my problem for a long time - the hashrate is full even after launching the miner on other algorithms. But there are nuances:
LHR was indeed unblocked by 74 - 73.5%, but the actual hash rate remained at the same level (35 mhs with lhr-mod-1 and 34.8 mhs with lhr-mod-2). If you manage to stay at 74% for a long time, then yes, the difference can still be noticed. Otherwise, you won't notice the difference.
![Снимок экрана (139)](https://user-images.githubusercontent.com/30767224/144767384-c5bfd5f2-1a30-4354-a0fe-942bd90c5703.png)
Moreover, lhr-mod-1 consumes more power than lhr-mod-2.
In total, there are two options for my 3060 lxr:
lhr-mod-1: 34.8 mhs for 95 watts
lhr-mod-2: 35.0 mhs per 118 watts
These + 0.2-0.5 mhs do not pay for themselves. In addition, the lhr-mod-2 has lower temperature, less wear on the card and less wear on its cooling. So if you're still thinking between lhr-mod-1 and lhr-mod-2, the choice is clear. lhr-mod-2 is your bro.
In my case, lxr-mod-2 may not start correctly after launching other algorithms, so I will use lhr-mod-1 for now.

**Reply by izccccc** [reply]
> 同样的问题在这里。使用 2x 3080 LHR，当 LHR unlok 工作正常时，我得到 71mh，但大多数时候它以 40mh 结束，即使控制台显示 LHR 模式为 71。
> 
> 最糟糕的是，NBminer 没有注意到低哈希率，而是继续使用 ethash 挖矿，而不是切换到其他算法。
40.1拒绝率高，换39.7就好了

**Comment by Ildar2015** [comment]
40.1 rx6600 20MH/S in 10 hours on pool
returned to 39.7 RX6600 28.8MH/S on pool

**Comment by wmm1337** [comment]
hi, i have a question. start_eth_overclock does the file fit the gpu setting into the windows or when opening bat file??

**Comment by m1k3b13n** [comment]
try to use the cmd that changes the down value from 0.5 (default) to 0.2 .
lock the core at 1600, mem +1300

**Comment by triviumfan** [comment]
all gpu setting (-cclock -mclock) dont back to defaults after closing miner.

**Comment by triviumfan** [comment]
hey.
my config is:
nbminer -a ethash -o stratum+tcp://ethash.poolbinance.com:8888 -u
triviumfan.both -log -lhr-mode 1 -cclock @***@***.*** -mclock 1300,1300

пт, 26 нояб. 2021 г. в 04:37, Jamie ***@***.***>:

**Comment by francisco-preto** [comment]
I am trying to have my core clock fixed in 1500 (--cclock @1500 ) but I can see a variation. Sometimes is 1500 but the next is 1340, and later 1450 and later 1500, ....
Question: Do you identify this as a problem?

**Reply by francisco-preto** [reply]
I have 3 3060 ti lhr (palit, leadteck and pny). I have the same problem to all 3 GPU´s

**Comment by miner8278** [comment]
請問設置核心跟記憶體超頻，有顯卡型號的限制嗎?我的1070TI不能設置

**Reply by miner8278** [reply]
![image](https://user-images.githubusercontent.com/32041410/143725384-bd5604b1-a1f7-4a20-b70d-91121287b42c.png)

**Reply by Tom5051** [reply]
10 series cards do not support setting the clock or memory speed through nbminer, even if you have 20 or 30 series cards too it will cause an error. You need to use a 3rd party app such as MSI Afterburner to set your OC settings if you have any 10 series cards in your farm.

**Comment by m1k3b13n** [comment]
@miner8278 you need to update to latest drivers, or close to.. I saw this
similar error using too old of drivers

**Reply by miner8278** [reply]
我嘗試了，但是依舊不行
![image](https://user-images.githubusercontent.com/32041410/143816808-4210237e-628c-4ec3-a249-af3ef9fd9934.png)

**Reply by miner8278** [reply]
不過記憶體超頻可以

**Comment by RGcomputacao** [comment]
How would it be to set the clocks of more than one gpu? Being different clocks and PL different between them? For example, I have a 3080ti, a 3080 and a 3060ti

**Reply by Tom5051** [reply]
use a comma between your values.

**Comment by gene292** [comment]
NBMiner 40.1 does not recognize amd rx 560 with 4Gig
I know it;s an old card, but I thought it might work for RVN.
Is this a compatibility issue?
Anything I can do to use it?

[12:02:18] INFO - ------------------- System -------------------
[12:02:18] INFO - OS:     Ubuntu 18.04.6 LTS, 4.15.0-162-generic
[12:02:18] INFO - CPU:    AMD FX(tm)-8320 Eight-Core Processor           
[12:02:18] INFO - RAM:    6416 MB / 15998 MB
[12:02:18] INFO - ------------------- Config -------------------
[12:02:18] INFO - ALGO:   kawpow
[12:02:18] INFO - URL:    stratum+tcp://stratum-ravencoin.flypool.org:13333
[12:02:18] INFO - USER:   RW7W1X392kJy2Hf3WwHVGaUcY56Hy6R2YN.default
[12:02:18] INFO - TEMP:   limit 90C, start 85C
[12:02:18] ERROR - Devices initialize failed.
[12:02:18] ERROR - No GPU available for mining.
[12:02:19] ERROR - !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
[12:02:19] ERROR - Mining program unexpected exit. 
[12:02:19] ERROR - Code: 255, Reason: Unknown error
[12:02:19] ERROR - Restart miner after 10 secs ... 
[12:02:19] ERROR - !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

**Comment by Sever72** [comment]
![2021-12-04_00-36-38](https://user-images.githubusercontent.com/95409595/144662678-da339875-7365-4267-bec8-bd89b07d22ba.png)
Подскажите в чем проблема?

**Reply by SDrag96** [reply]
I am also getting the same issue...anyone can help with a solution for the problem?

**Comment by Kcasstor** [comment]
Can someone help me? I'm trying to install NBminder on windows 11, but the following error message pops up: "nbminder is not recognized as an external or internal command, an operable program or a batch file"

**Comment by izccccc** [comment]
40.1拒绝率太高了

**Comment by Kcasstor** [comment]
Just disable Firewall

________________________________
De: SDrag96 ***@***.***>
Enviado: domingo, 9 de janeiro de 2022 08:36
Para: NebuTech/NBMiner ***@***.***>
Cc: Kcasstor ***@***.***>; Comment ***@***.***>
Assunto: Re: [NebuTech/NBMiner] NBMiner v40.1 (Discussion #657)


I am also getting the same issue...anyone can help with a solution for the problem?

—
Reply to this email directly, view it on GitHub<https://github.com/NebuTech/NBMiner/discussions/657#discussioncomment-1934827>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AW7EPDXV3SRLPZQVWHAMMTLUVFXKNANCNFSM5IU6BGCQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by YuFeng-ovo** [comment]
-power-limit, -pl: Set power limitation of GPU

why it cant work，i use this
nbminer -a ethash -o stratum+tcp://eth.ss.poolin.me:443 -pl 75%% -u xxx -log
But my GPU usage is still 100%

Can anyone help me, much appreciated!

i try "--pl 35%%" and "--power-limit" still cant work
please, anyone

**Comment by Amishtek** [comment]
Can ANYONE advise what would cause my GPU (gigabyte gaming oc 3080 nonlhr) to HANG "ERROR - GPU 1 hung detected!,  ERROR - Code: -1073741819, Reason: Process crashed", prior to this error NBMiner threw another exception " CUDA Error: unknown error (err_no=30) ERROR - Code: 1, Reason: Unknown error"  --- the miner self fixes and restarts after a while but I would like to know what I should do. I am running the miner on my windows PC.  core is 0, core clock -150, memory clock +850, PL 65, TL 75, fan speed 80.  I have reduced the memory clock and core clock already but still get the message. seeking community guidance on optional things I should try.

**Reply by Amishtek** [reply]
still crashing any assistance would be greatly appreciated.

**Reply by Tom5051** [reply]
> Can ANYONE advise what would cause my GPU (gigabyte gaming oc 3080 nonlhr) to HANG "ERROR - GPU 1 hung detected!, ERROR - Code: -1073741819, Reason: Process crashed", prior to this error NBMiner threw another exception " CUDA Error: unknown error (err_no=30) ERROR - Code: 1, Reason: Unknown error" --- the miner self fixes and restarts after a while but I would like to know what I should do. I am running the miner on my windows PC. core is 0, core clock -150, memory clock +850, PL 65, TL 75, fan speed 80. I have reduced the memory clock and core clock already but still get the message. seeking community guidance on optional things I should try.

lower your memory clock or keep the card cooler.

**Comment by Tabiin-77** [comment]
Hello

**Comment by mj-malabi** [comment]
Im receiving error. What should I do?
<img width="670" alt="image" src="https://user-images.githubusercontent.com/101132997/157088175-4e2e2946-99b4-4c78-a6b5-a22371e16dd4.png">

**Reply by Tom5051** [reply]
> Im receiving error. What should I do? <img alt="image" width="670" src="https://user-images.githubusercontent.com/101132997/157088175-4e2e2946-99b4-4c78-a6b5-a22371e16dd4.png">



> <img alt="image" width="670" src="https://user-images.githubusercontent.com/101132997/157088175-4e2e2946-99b4-4c78-a6b5-a22371e16dd4.png">

you need a GPU with 6GB or more to mine Ethereum. There are no digital assets that can be mined with a 2gb card sorry.

**Comment by Alexlnt** [comment]
Hi 
I'm trying to instal drivers but it show me an error. " MEMORY TWEAKER DRIVER INSTAL FAILED 2" 
can sombody help me ?

**Reply by Tom5051** [reply]
> Hi I'm trying to instal drivers but it show me an error. " MEMORY TWEAKER DRIVER INSTAL FAILED 2" can sombody help me ?

what OS, what GPU and what driver version. This happens on my laptop which has a 1070 in it.

**Comment by nbnslider** [comment]
Where are the new versions? Why is there nothing?

**Comment by ph4ntom123-3** [comment]
hi everyone. i've just started mining and have one question. i'm using NiceHash to mine in NBminer 40,1, what is that EFF/WATT 
higher = better ,or less = better. 
got about 55 Mh on my 3060ti LHR OC, 65% PL = 155W, 65% fan, 40 temp. on Octopus 
![Untitled](https://user-images.githubusercontent.com/70589347/162882919-0b184b91-a664-4124-aa25-5b378f299ca7.png)

got about 46 Mh, 52% PL= 123W, 65% FAN = 35C , DaggerHashimoto 


![dagger](https://user-images.githubusercontent.com/70589347/163405685-9ee6d3b3-437e-4094-8fc3-63a1338829d9.jpg)

T-rex 0.25.12   in Nicehash






![T-rex](https://user-images.githubusercontent.com/70589347/163683157-1e06bdaa-8ca4-41a5-94d5-32b1781c5575.jpg)


thanks a lot for any replies

**Reply by Tom5051** [reply]
Eff/Watt - higher value is better.

**Comment by hernanJardon** [comment]
Any LHR update?

**Comment by ph4ntom123-3** [comment]
T-rex in NiceHash

![01](https://user-images.githubusercontent.com/70589347/168285544-58778ceb-c554-4ca9-aa46-5862da468223.jpg)
